user-guide/modules/ROOT/pages/reporting-issues.adoc

```diff
@@ -6,37 +6,24 @@
 
 == Reporting Bugs
 
-1. If you know where to look in the source code, make sure
- the bug isn't already fixed in the latest sources. The most
- recent version of everything on the Boost web site is
- available from the https://github.com/boostorg/wiki/wiki/Getting-Started:[git repositories].
-2. https://github.com/boostorg/:[Search the issues]
- on GitHub to make sure we don't already know about the bug.
- If we do, you can add further information to an existing bug ticket.
-3. https://github.com/boostorg/:[Create a new issues]
- in the repository of the particular library of your interest.
+. If you know where to look in the source code, make sure the bug isn't already fixed in the latest sources. The most  recent version of everything on the Boost web site is available from the https://github.com/boostorg/wiki/wiki/Getting-Started:[git repositories].
```

Comment:
Most subsections in "Common Scenarios" are kind of weird. Ignoring other subsections for now, I think this one could be under "Community". I can't see a person who just got started reporting issues. Neither we want to incentivize that.

Reply:
I will move to Community.

Reply:
Idk if this is thw story for it, but I think the wiki is super unmaintained so I'd refrain from linking to it.

Reply:
Oh... I didn't even notice the link. These two sentences seem unrelated and out of place.

> If you know where to look in the source code, make sure the bug isn't already fixed in the latest sources. 

This first sentence is about reporting issues. And yes. The real and de facto place to report issues nowadays is the corresponding github repository. It's the place where the library author will get a notification and is more likely to reply unless it's boost.asio in which case you're out of luck anyway :laughing:. A second best place is the mailing list, but it's not 1% as good as the github repository in my opinion. Issues in the ML have a tragedy of the commons in terms of replies and are even somewhat annoying sometimes.

> The most recent version of everything on the Boost web site is available from the https://github.com/boostorg/wiki/wiki/Getting-Started:[git repositories].

So this second sentence is about something else entirely. For now, we could just remove this sentence because it's simply false. Once our (initial) work is done with site-docs, the website content will come from whatever becomes of this very repository and cppalliance/temp-site.